### DISCUSSION TITLE
Is it possible to export SwiftGodot projects for the web?

### DISCUSSION THREAD

**Discussion by DavidPD** [migueldeicaza/SwiftGodot#446] [Q&A] [answered]
I'm super interested in using a more type-safe/null-safe language than GDScript for game dev (C#, Swift, Rust). But when doing game jams I place a lot of value on being able to run the game in a browser. I've seen some Swift WASM support, but can't find anything about SwiftGodot specifically.

**Comment by migueldeicaza** [comment, marked as answer]
The short version is that currently this is not a scenario that works.

The longer story is:

There has been some discussion on what is involved in our Slack group, I am including it here:

https://swiftongodot.slack.com/archives/C05BYJ95S5D/p1708199640584689

The short version is that there is some work needed in Swift and Wasm (not sure where) to support dynamic libraries before we can have a smooth experience.   The challenge is described here:

https://discord.com/channels/780838335798706197/790246707648135229/1165758283320148019

I just asked a follow up question about it, as there might be a way out:

https://discord.com/channels/780838335798706197/790246707648135229/1225627645765484655

**Comment by DavidPD** [comment]
Thanks for looking into this! I appreciate it's a tough problem as not even the official C# works with web. I'll just keep keeping my eyes out!

**Comment by migueldeicaza** [comment]
Will be tracking this feature here:

https://github.com/migueldeicaza/SwiftGodot/issues/447
